.github/workflows/release.yml

```diff
@@ -0,0 +1,106 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      publishPreRelease:
+        description: 'Publish a pre-release ?'
+        required: true
+        type: choice
+        options:
+          - 'true'
+          - 'false'
+        default: 'false'
+      publishToMarketPlace:
+        description: 'Publish to VS Code Marketplace ?'
+        required: true
+        type: choice
+        options:
+          - 'true'
+          - 'false'
+        default: 'true'
+      publishToOVSX:
+        description: 'Publish to OpenVSX Registry ?'
+        required: true
+        type: choice
+        options:
+          - 'true'
+          - 'false'
+        default: 'true'
+
+jobs:
+  packaging-job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout vscode-yaml
+        uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          path: release
+          node-version: 20
+      - name: Install dependencies
+        run: |
+          npm install -g typescript "yarn" "@vscode/vsce" "ovsx"
+          npm install
```

Comment:
Given this project uses yarn, we should use yarn to install dependencies. In the next step, on line 55, you run yarn install, so I think it should be okay to remove this line (just line 45, we still need line 44).